htdocs/compta/tva/index.php

```diff
@@ -43,61 +44,89 @@
 	$conf->global->SOCIETE_FISCAL_MONTH_START = 1;
 }
 
-// Date range
-$year = GETPOST("year", "int");
-if (empty($year)) {
-	$year_current = $current_date['year'];
-	$year_start = $year_current;
-} else {
-	$year_current = $year;
-	$year_start = $year;
-}
-$date_start = dol_mktime(0, 0, 0, GETPOST("date_startmonth"), GETPOST("date_startday"), GETPOST("date_startyear"));
-$date_end = dol_mktime(23, 59, 59, GETPOST("date_endmonth"), GETPOST("date_endday"), GETPOST("date_endyear"));
-// Set default period if not defined
-if (empty($date_start) || empty($date_end)) { // We define date_start and date_end
-	$q = GETPOST("q", "int");
-	if (empty($q)) {
-		if (GETPOST("month", "int")) {
-			$date_start = dol_get_first_day($year_start, GETPOST("month", "int"), false);
-			$date_end = dol_get_last_day($year_start, GETPOST("month", "int"), false);
+$refresh = GETPOSTISSET('submit') ? true : false;
+
+if ($refresh === false) {
+	$year_current = intval(strftime('%Y', $now));
```

Comment:
strftime is forbidden (TZ difference between server and user are not correctly managed). Can you replace with dol_print_date($now, '%Y')

Reply:
Noted